warden/app/oracle.go

```diff
@@ -238,6 +240,34 @@ func createSlinkyUpgrader(app *App) AppUpgrade {
 			if err != nil {
 				return nil, fmt.Errorf("failed to set x/marketmap params: %w", err)
 			}
+
+			err = app.EvmKeeper.SetParams(
```

Comment:
this is inside the v0.4 migration code (already executed in buenavista), so this code probably don't belong here

Reply:
Removed upgrade part at all as we plan to start this version from scratch